enterprise/constants.py

```diff
@@ -93,6 +93,7 @@
 
 ENTERPRISE_LEARNER_ROLE = 'enterprise_learner'
 ENTERPRISE_ADMIN_ROLE = 'enterprise_admin'
+ENTERPRISE_OPERATOR_ROLE = 'enterprise_operator'
```

Comment:
i might rename the value of this constant to something like `enterprise_openedx_operator`, just to distinguish it further in meaning from `enterprise_admin`